tools/rebuild.sh

```diff
@@ -75,25 +75,7 @@ fi
 : "${SGX_SDK?Missing environment variable SGX_SDK}"
 : "${PKG_CONFIG_PATH?Missing environment variable PKG_CONFIG_PATH}"
 
-# Set proxy for Intel Architectural Enclave Service Manager (AESM) and start
-if [[ ${SGX_MODE} &&  "${SGX_MODE}" == "HW" ]]; then
-
-    # Add proxy settings, if proxy is present and not set
-    grep -qs "^proxy type" /etc/aesmd.conf
-    if [ $? -ne 0 -a -n "$http_proxy" ] ; then
-       echo "Setting AESMD proxy type"
-       echo "proxy type = manual" >> /etc/aesmd.conf
-    fi
-    grep -qs "^aesm proxy" /etc/aesmd.conf
-    if [ $? -ne 0 -a -n "$http_proxy" ] ; then
-       echo "Setting AESMD proxy"
-       echo "aesm proxy = $http_proxy" >> /etc/aesmd.conf
-    fi
-
-    # Starting aesm service
-    echo "Starting aesm service"
-    /opt/intel/libsgx-enclave-common/aesm/aesm_service &
-else
+if [ -z ${SGX_MODE} ]; then
```

Comment:
Is it ok to continue if SGX_MODE is set to anything else other than `HW` or `SIM`?

Reply:
no modes other than SIM or HW should go through. I will add the check.